sdk/personalizer/azure-ai-personalizer/src/main/java/com/azure/ai/personalizer/PersonalizerAudience.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.personalizer;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+import java.util.Collection;
+
+/** Defines values for PersonalizerAudience. */
+public class PersonalizerAudience extends ExpandableStringEnum<PersonalizerAudience> {
```

Comment:
You may not need this now. This is to support multi-tenant cloud support for the SDK.

Reply:
We do need to support gov cloud. I removed China from this list of supported options since we don't support ChinaCloud.